make/Images.gmk

```diff
@@ -43,8 +43,9 @@ ALL_MODULES := $(call FindAllModules)
 
 $(eval $(call ReadImportMetaData))
 
+# SapMachine 2021-09-24 : add jcmd to JRE
 JRE_MODULES += $(filter $(ALL_MODULES), $(BOOT_MODULES) \
-    $(PLATFORM_MODULES) jdk.jdwp.agent)
+    $(PLATFORM_MODULES) jdk.jdwp.agent jdk.jcmd jdk.internal.jvmstat)
```

Comment:
I think jdk.internal.jvmstat can/should be left out because it is a required module of jdk.jcmd so it should automatically be resolved... Did you try this? If not, please do :)

Reply:
Hi Christoph, looks like jdk.internal.jvmstat  can be removed indeed , I tried it and jcmd still works nicely.